### PR TITLE
Minor Viewer3D improvements

### PR DESCRIPTION
**Pull request by dcoeurjo**
- Default background is lighter
- The light position has been changed so that it is not axis aligned anymore 
- default window size is 800x600

### CODE REVIEW

**Comment by dcoeurjo** [comment]
BTW, I don't understand why ```viewer3D-5-custom``` points have black edges..

File: src/DGtal/io/viewers/Viewer3D.ih [hunk 1]
```diff
@@ -820,7 +820,7 @@ DGtal::Viewer3D<Space, KSpace>::init()
   myGLScaleFactorY=1.0;
   myGLScaleFactorZ=1.0;
   myGLLineMinWidth=1.5;
-
+  setBackgroundColor ( QColor ( 217, 228, 255 ) );
```
Comment (kerautret) [inline src/DGtal/io/viewers/Viewer3D.ih]: For the color ok but the dark one is no more present. 
We can perhaps put the new one by default (instead white) and let the dark one associated to Key_B ?

**Comment by kerautret** [comment]
else looks fine thanks

File: src/DGtal/io/viewers/Viewer3D.ih (hunk 1 above)
Comment (dcoeurjo) [inline src/DGtal/io/viewers/Viewer3D.ih]: The white background is still needed for exports and snapshots. 
The idea of this PR is to replace the dark background (which is not great when displaying isothetic quads with classical lightning) by a "lighter" one.

**Comment by dcoeurjo** [comment]
Does the resize to 800x600 works on your machine?

File: src/DGtal/io/viewers/Viewer3D.ih (hunk 1 above)
Comment (kerautret) [inline src/DGtal/io/viewers/Viewer3D.ih]: ok I see and we could later add a third 'dark' mode if needed.

**Comment by kerautret** [comment]
yes it works, not you ? (at least for the examples)

**Comment by dcoeurjo** [comment]
not always working.. strange...
Ready to merge?

**Comment by jlevallois** [comment]
It's ok for me

**Comment by dcoeurjo** [comment]
ok merging. thx

**Pull request merged**